public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -0,0 +1,23 @@
+library pipes_examples.exponential_strength_pipe;
+
+import 'package:angular2/angular2.dart';
+import 'dart:math';
+
+/*
+* Raise the value exponentially
+* Takes a value that defaults to 0 and an exponent argument that defaults to 1.
+* Checks for value to be a string or number.
+* Usage:
+*   value | exponentialStrength:exponent
+* Example:
+*   {{ 2 |  exponentialStrength:10}}
+*   formats to: 1024
+*/
+
+@Pipe(name: 'exponentialStrength')
+@Injectable()
+class ExponentialStrengthPipe {
+  transform(dynamic value, [List<dynamic> args]) => pow(
+      int.parse(value.toString(), onError: (source) => 0),
```

Comment:
Break this into a "full" function. Create vars for each argument to pow and then `return pow(a, b);`

It'll be easier to read, me thinks